extensions/objects/templates/image-editor.component.ts

```diff
@@ -68,8 +68,12 @@ import { WindowRef } from 'services/contextual/window-ref.service';
 const gifshot = require('gifshot');
 import * as gifFrames from 'gif-frames';
 
+// When attach GifFrames to the window and use it in our codebase and the
```

Comment:
We attach GifFrames to the window and use it in our codebase, but the default Window ...


(Also you might want to add a TODO explaining the context in which this was added and that we should get rid of it because we should not be declaring globals.)

Reply:
Done!